docs-chef-io/content/inspec/install.md

```diff
@@ -21,12 +21,14 @@ To see which platforms and platform versions Chef InSpec is supported on, see th
 
 You can install Chef InSpec using a curl script.
 
-{{< note >}}Please follow steps mentioned in this document [Chef Software Install Script](/install_omnibus/). Please replace `<LICENSE_ID>` with your licenseId.{{< /note >}}
-
 ```bash
 curl -L https://chefdownload-commerical.chef.io/install.sh?license_id=<LICENSE_ID> | sudo bash -s -- -P inspec
 ```
 
+Replace `<LICENSE_ID>` with your license ID.
+
+For more information about the install script, see the [Chef Install Script documentation](/chef_install_script/).
```

Comment:
So install script is currently giving a 404 error so that is expected right @IanMadd

Reply:
@Vasu1105 
Where is it sending you? 
It works for me --> https://deploy-preview-7144--chef-inspec.netlify.app/chef_install_script/

Reply:
It's landing me here https://github.com/inspec/inspec/blob/d2e6dc27791ecca78d237650935a4cb780182945/chef_install_script

Reply:
The chef_install_script page comes from another repo, so it won't appear here. 
Try here https://deploy-preview-7144--chef-inspec.netlify.app/inspec/install/

Reply:
Ok. Merging the PR. Thanks!!